### PR TITLE
CHAOS-5107 Update tutorial of chaos experiment from a blank canvas

### PR DESCRIPTION
**Pull request by SmritiSatya**
[Update tutorial: "chaos experiment from a blank canvas"](https://661f6001af8e4acb6e58267d--harness-developer.netlify.app/docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas)

### CODE REVIEW

**Review by neelanjan00 — DISMISSED** [review]

File: docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas.md [hunk 1]
```diff
@@ -1,56 +1,132 @@
...
 sidebar_position: 3
 redirect_from:
   - /tutorials/chaos-experiments/chaos-experiment-from-blank-canvas
 ---
 
-This tutorial guides you through creating chaos experiments from scratch. Here, you'll construct a chaos experiment from scratch in the Chaos Studio.
+In this tutorial, you will create chaos experiments from scratch and execute them on the sample boutique application.
 
-## Constructing a chaos experiment from scratch
+This experiment follows the same steps of [your first chaos experiment](/docs/chaos-engineering/get-started/tutorials/first-chaos-engineering.md) by creating a chaos environment, and infrastructure and using the same boutique application by targeting the pods of the `cart` microservice, except the way you create an experiment. Creating a chaos experiment involves the following steps:
```
Comment (brian-f-harness) [inline docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas.md:11]:
```suggestion
This experiment follows the same steps of [your first chaos experiment](/docs/chaos-engineering/get-started/tutorials/first-chaos-engineering.md) by creating a chaos environment and infrastructure and using the same boutique application by targeting the pods of the `cart` microservice, except the way you create an experiment. Creating a chaos experiment involves the following steps:
```

File: docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas.md [hunk 2]
```diff
@@ -1,56 +1,132 @@
...
+* [Prerequisites to execute chaos experiments](/docs/chaos-engineering/get-started/tutorials/prerequisites.md)
+* [Execute your first chaos experiment using a template](/docs/chaos-engineering/get-started/tutorials/first-chaos-engineering.md)
+
+### Step 1: Set up environment and infrastructure
+
+1. Follow the steps to set up [your environment](/docs/chaos-engineering/get-started/tutorials/first-chaos-engineering#step-2-add-a-chaos-environment) and [chaos infrastructure](/docs/chaos-engineering/get-started/tutorials/first-chaos-engineering#step-3-add-a-chaos-infrastructure).
+
+### Step 2: Construct a chaos experiment from a blank canvas
+
+2. Once you have your environment and chaos infrastructure in place, you can create a chaos experiment. To create an experiment, navigate to **Chaos Experiments** in the sidebar menu and click **New Experiment**.
```
Comment (brian-f-harness) [inline docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas.md:32]:
```suggestion
2. Once you have your environment and chaos infrastructure in place, you can create a chaos experiment. To create an experiment, navigate to **Chaos Experiments** in the left nav, and then select **New Experiment**.
```

File: docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas.md [hunk 3]
```diff
@@ -1,56 +1,132 @@
...
+
+![create new experiment](./static/first-chaos/create-new-experiment-1.png)
+
+3. Specify the experiment name and a description (optional) and tags (optional). Choose the target infrastructure, click **Apply**, and click **Next**.
```
Comment (brian-f-harness) [inline docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas.md:36]:
```suggestion
3. Specify the experiment name, description (optional), and tags (optional). Choose the target infrastructure, click **Apply**, and click **Next**.
```

File: docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas.md [hunk 4]
```diff
@@ -1,56 +1,132 @@
...
+
+![specify parameters](./static/first-chaos/specify-params-2.png)
+
+4. In the Experiment Builder, select **Blank Canvas** and click **Start with blank canvas**.
+
+![blank canvas](./static/chaos-experiment-from-blank-canvas/blank-canvas.png)
 
-To create the chaos experiment, go to **Chaos Experiments** in the sidebar menu and choose **New Experiment**. Then, add the experiment name and optionally a description and tags. Then, choose the target infrastructure, which we created previously. Choose **Next**. In the Experiment Builder, choose **Blank Canvas** and select **Start with blank canvas**.
+5. This will lead you to the page where you can add chaos faults to your experiment. Click **Add**.
```
Comment (brian-f-harness) [inline docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas.md:44]:
```suggestion
5. This opens the page where you can add chaos faults to your experiment. Click **Add**.
```

File: docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas.md [hunk 5]
```diff
@@ -1,56 +1,132 @@
...
+7. Add details in the **Target Application** tab, such as the `hce` namespace, `app=cartservice` label, and application kind `deployment`.
 
-![Specify Target Application](./static/chaos-experiment-from-blank-canvas/specify-target-application.png)
+![target app](./static/chaos-experiment-from-blank-canvas/target-app.png)
 
-Then, choose the **Tune Fault** tab to configure the fault parameters. Here, we will define the fault execution duration to be 30 seconds, with an interval of 10 seconds, so that in every 10 seconds we will be deleting the cart deployment pods for a total of 30 second duration. Lastly we have specified the pod affected percentage to be 50% so that half of all the deployment pods and at minimum one pod will be deleted in each iteration of chaos.
+8. In the **Tune Fault** tab, specify **total chaos duration** as 30, **chaos interval** as 5, **pods affected perc** as 50, and the default weight to 10.
```
Comment (brian-f-harness) [inline docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas.md:56]: Are these UI elements all lowercase? If not, suggest matching the case in the UI.

File: docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas.md [hunk 6]
```diff
@@ -1,56 +1,132 @@
...
 
 ![Tune Fault](./static/chaos-experiment-from-blank-canvas/tune-fault.png)
 
-Lastly, choose the **Probes** tab. Here, we have a default application **health-check** probe defined which validates the target application health before and after the fault injection to ensure that the application is unharmed due to the experiment. Let us add one more probe so that we can validate the carts microservice health. Select **Deploy new Probe**, ensure that the probe type is `HTTP Probe`, select **Continue** and then again select **Continue**. Now, we need to provide the HTTP probe URL so that the probe can make GET requests at that URL and check if the status code in response is 200 for the successful validation of the probe. Provide the URL `http://frontend/cart`, which is the URL for the cart page. Also, reduce the response timeout to 5 seconds. Then, select **Setup the Probe**.
+:::tip
+The `pods affected perc` describes the percentage of pods that are affected due to the unexpected pod deletion. A minimum of one pod is deleted.
```
Comment (brian-f-harness) [inline docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas.md:61]:
```suggestion
The **pods affected perc** describes the percentage of pods that are affected due to the unexpected pod deletion. A minimum of one pod is deleted.
```

File: docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas.md [hunk 7]
```diff
@@ -1,56 +1,132 @@
...
+### Step 3: Create a new HTTP probe
+8. In the **Probes** tab, you can create a new probe. For this, click **Select or Add new probes**.
+
+![Probes Config](./static/first-chaos/probes-config.png)
+
+9. Select **New probe**.
+
+![new probe](./static/chaos-experiment-from-blank-canvas/new-probe.png)
+
+10. Select **Kubernetes** as the infrastructure type, and **HTTP** as the probe type. This leads you to the probe configuration screen.
```
Comment (brian-f-harness) [inline docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas.md:73]:
```suggestion
10. Select **Kubernetes** as the infrastructure type, and **HTTP** as the probe type. This opens the probe configuration screen.
```

Comment (brian-f-harness) [reply docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas.md:73]: Suggest matching the case of the UI. Is it probe configuration?

File: docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas.md [hunk 8]
```diff
@@ -1,56 +1,132 @@
...
+![add to fault](./static/chaos-experiment-from-blank-canvas/add-to-fault.png)
 
-When ready, start the experiment execution by selecting **Run** on the top right corner of the screen. You'll be able to observe the experiment added to the list of chaos experiments and it should be in a `Running` status. Choose **Current Execution** to get a detailed view.
+16. Select mode as **Continuous** and click **Apply changes**.
 
-![Experiment Executing](./static/chaos-experiment-from-blank-canvas/experiment-executing.png)
+![continuous](./static/chaos-experiment-from-blank-canvas/continuous-mode.png)
 
-Once the fault is running, we can check for the detailed view of the experiment. We can follow the logs of the experiment run as it gets executed. 
+17. Click **Apply changes** again to confirm all parameters configuration. Click 'X' to close the overlay modal. Click **Apply changes** again.
```
Comment (brian-f-harness) [inline docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas.md:101]:
```suggestion
17. Click **Apply changes** again to confirm all parameters configuration. Click **X** to close the overlay modal. Click **Apply changes** again.
```

File: docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas.md [hunk 9]
```diff
@@ -1,56 +1,132 @@
...
-At the same time, we can also check for the status of the cart deployment pod. Upon executing the following command you will get a similar output. It is evident that the Pod Delete fault has caused the cart pod to be terminated and a new pod has recently replaced it, for whose container is yet to be created.
+### Step 4: Verify cart page health
+
+18. Before executing the experiment, verify that the cart page is healthy and accessible from the front end, as seen at the `/cart` route.
+
+![Online Boutique App Cart](./static/chaos-experiment-from-blank-canvas/online-boutique-app-cart.png)
+
+### Step 5: Observe chaos execution
+
+19. When all the prerequisites are fulfilled and parameters are set, you can start the experiment execution by selecting **Run** on the top right corner of the screen.
```
Comment (brian-f-harness) [inline docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas.md:113]:
```suggestion
19. When all the prerequisites are fulfilled and parameters are set, you can start the experiment execution by selecting **Run** .
```

Comment (brian-f-harness) [reply docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas.md:113]: Recommend avoiding directional language when possible.
https://developers.google.com/style/translation#general-dos-and-donts

File: docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas.md [hunk 10]
```diff
@@ -1,56 +1,132 @@
...
+
+![Run and save](./static/first-chaos/run-n-save.png)
+
+20. You can see that once you click **Run**, an experiment run is scheduled. You can see the status of every step in the tab.
```
Comment (brian-f-harness) [inline docs/chaos-engineering/get-started/tutorials/chaos-experiment-from-blank-canvas.md:117]: This isn't a step. Indent so the content is inline with the steps? Make it an info note?

**Review by brian-f-harness — APPROVED** [review]
Left a few minor suggestions. LGTM otherwise.

**Pull request merged**